src/js/components/campaignpage/Charities.jsx

```diff
@@ -104,11 +103,49 @@ const Charities = ({ charities, donation4charity }) => {
 				{sogiveCharitiesWithDonations.map((charity, i) =>
 					<CharityCard i={i} key={charity.id} charity={charity} donationValue={getDonation(charity)} />
 				)}
+				<Details charities={sogiveCharities}/>
 			</Container>
 		</div>
 	);
 };
 
+// Extra smallprint details for charities
+const Details = ({charities}) => {
+	
+	const [open, setOpen] = useState(false);
```

Comment:
I think - Let's have them always visible. That feels better for legal small print. And it's the bottom of the page, so it doesn't harm anything.